### PR TITLE
Change the README

### PR DESCRIPTION
**Pull request by faraznaeem**
### User story
```
As a Coach 
In order to teach the cohort to create a PR
I would like to demo how to create a PR
```
### What I have learned:
- Create PR
- Push to Origin
- difference between base and head fork

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,8 @@
-# git_demo_2018
-Forking, Pushing and PRs
+# This is my new changes to the README
+
+- In this Demo I will
+- Fork a new repo
```
Comment (SophieSjoberg) [inline README.md]: Why so serious?

**Review by SophieSjoberg — CHANGES_REQUESTED** [review]
Please be less serious in the future

File: README.md (hunk 1 above)
Comment (faraznaeem) [reply README.md]: Well its soon time to go home and im kinda hungry, but i will make the changes!

**Review by SophieSjoberg — APPROVED** [review]

**Comment by SophieSjoberg** [comment]
Good job!

**Pull request merged**